### DISCUSSION TITLE
Wrong sonde serial number for M10 probes

### DISCUSSION THREAD

**Discussion by DadoRdz** [Eshco93/dxlAPRS-SHUE#3] [General]
Dear Simon,
We noticed a possible error in the operation of the SHUE script for the M10 Meteomodem probe. For example for the probe that was released from Belgrade today (Sep. 20) under the tag "201-2-12655" (ME2122A5F), SHUE changes this tag to "001---2-12655" when sending the JSON package to Sondehub. The same erronous code appears in the log file name as in the actual CSV file for that probe in the "serial" column. Sondehub recognizes it as a new balloon because of the different label, so two balloons appear on the map, the one with the right label, as well as the fake one with the wrong label, which causes big problems. It would be appreciated if this error could be fixed, because otherwise the program is excellent for all dxlAPRS users.

**Comment by Eshco93** [comment]
Hey :)
This is a known problem. Please take a look at this issue:

https://github.com/Eshco93/dxlAPRS-SHUE/issues/2

In my reply there, I explained the problem(s) in all details.
Unfortunately, I'm not able to completely fix this in dxlAPRS-SHUE. There is a fundamental problem in the latest version of sondemod.

**Reply by Eshco93** [reply]
Okay, I did a bit of research on this. Turns out that I can actually implement a workaround for this sondemod-related issue in dxlAPRS-SHUE.
I can use the "APRS serial" of the radiosonde - ME2122A5F in your example - and calculate the actual serial for SondeHub from that.
I'll soon provide an update for dxlAPRS-SHUE.

**Reply by DadoRdz** [reply]
Thank you very much. Right after I put this post in Discussions, I realized I put it in the wrong place and should have posted it in Issues and not here, but it was too late. Of course, there (in Issues) I immediately noticed that an identical problem had already been reported, i.e. that the problem is in the new version of sondemod, not in SHUE, and just such a newer version is necessary for JSON output. The idea with the APRS serial is excellent, so we eagerly await the new version of SHUE with this tricky patch.

**Reply by Eshco93** [reply]
The new version is released. This should be fixed now.

**Reply by DadoRdz** [reply]
Excellent, I'll give it a try.
